src/transformers/models/maskformer/convert_maskformer_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -0,0 +1,726 @@
+# coding=utf-8
+# Copyright 2022 Facebook AI Research and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+from argparse import ArgumentParser
+from dataclasses import dataclass
+from pathlib import Path
+from pprint import pformat
+from typing import Any, Dict, Iterator, List, Set, Tuple
+
+import torch
+import torchvision.transforms as T
+from PIL import Image
+from torch import Tensor, nn
+
+import requests
+from detectron2.checkpoint import DetectionCheckpointer
+from detectron2.config import get_cfg
+from detectron2.data import MetadataCatalog
+from detectron2.projects.deeplab import add_deeplab_config
+from transformers.models.maskformer.feature_extraction_maskformer import MaskFormerFeatureExtractor
```

Comment:
We usually use the feature extractor to prepare images for the model and to compare logits.

Is `torchvision.transforms as T` necessary?

Reply:
My thinking was to make the conversion script not depend on the `FeatureExtractor` to be sure that in the case of not equal logits, the bug is only in the conversions functions and not in the extractor